### DISCUSSION TITLE
Trying to hide icon from Tabs

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#6131] [Q&A] [answered]
I would like to hide the `icon` from the metadata in the tab of the documentation, but not in other parts of the page (i.e. left-side navigation).

Thing is, that I cannot figure out which partial is responsible for it.
I tried `nav_item.html` but this one seems to only affect the left-side navigation. I then tried `tabs_item.html` but this one doesn't seem to change anything?

I tried this setup. See Line 3 for the edit:
```jinja2
  {% macro render_content(nav_item, ref = nav_item) %}
    {% if nav_item == ref or "navigation.indexes" in features %}
      {% if nav_item.is_index and nav_item.meta.icon and not nav_item.meta.hide_icon %}
        {% include ".icons/" ~ nav_item.meta.icon ~ ".svg" %}
      {% endif %}
    {% endif %}
    {{ ref.title }}
  {% endmacro %}
  {% macro render(nav_item, ref = nav_item) %}
    {% set class = "md-tabs__item" %}
    {% if ref.active %}
      {% set class = class ~ " md-tabs__item--active" %}
    {% endif %}
    {% if nav_item.children %}
      {% set first = nav_item.children | first %}
      {% if first.children %}
        {{ render(first, ref) }}
      {% else %}
        <li class="{{ class }}">
          <a href="{{ first.url | url }}" class="md-tabs__link">
            {{ render_content(first, ref) }}
          </a>
        </li>
      {% endif %}
    {% else %}
      <li class="{{ class }}">
        <a href="{{ nav_item.url | url }}" class="md-tabs__link">
          {{ render_content(nav_item) }}
        </a>
      </li>
    {% endif %}
  {% endmacro %}
```

Also, before asking:

- Yes, I enabled theme extending in the mkdocs.yml file
- Yes, I named the file properly and put it in the right directory in the `custom_dir` directory.

Any ideas why it doesn't work? Am I editing the wrong partial?

**Comment by alexvoss** [comment]
`tabs-item.html` *should* be the right place. Can I look at your project or can you do a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/)?

**Reply by Andre601** [reply]
The project is here: https://github.com/Andre601/DecentHolograms-Wiki/tree/feature/move-to-mkdocs

The file is located here:
https://github.com/Andre601/DecentHolograms-Wiki/blob/feature/move-to-mkdocs/theme/partials/tabs_item.html

**Reply by kamilkrzyskow** [reply, marked as answer]
@Andre601 Without running the project, I see that you named the partial as `tabs_item.html` instead of `tabs-item.html`
EDIT: I made a typo myself while writing the message 😳

**Reply by Andre601** [reply]
Typical andre issue 😓 

I'm just used to using `_` because apparently `-` is bad in jinja2... Works now. Thanks.

**Reply by kamilkrzyskow** [reply]
I often make similar mistakes, so don't feel bad. Also it's not exactly bad in Jinja2, it's just that most programming languages don't allow `-` in the variable names, so the `_` is enforced in all Python, Jinja2 or even TypeScript scripts. As for the partial name it's just the convention that squidfunk used.

**Comment by alexvoss** [comment]
Thanks, @kamilkrzyskow, for applying your eagle eyes!
